contracts/RelayHub.sol

```diff
@@ -285,16 +285,15 @@ contract RelayHub is IRelayHub {
         uint256 toBurn = SafeMath.div(amount, 2);
         uint256 reward = SafeMath.sub(amount, toBurn);
 
-        require(relayData[relayManager].manager != address(0), "Relay is not registered");
```

Comment:
this was removed because the old code did not enforce this check, so the refactor should not either